ci/build/build-release.sh

```diff
@@ -77,7 +77,12 @@ EOF
 
 bundle_vscode() {
   mkdir -p "$VSCODE_OUT_PATH"
-  rsync ./lib/vscode-reh-web-*/ "$VSCODE_OUT_PATH"
+
+  # - Some extensions have a .gitignore which excludes their built source from
+  #   the npm package so exclude any .gitignore files.
+  # - Exclude Node as we will add it ourselves for the standalone and will not
+  #   need it for the npm package.
+  rsync -avh --exclude .gitignore --exclude /node ./lib/vscode-reh-web-*/ "$VSCODE_OUT_PATH"
```

Comment:
Will this fix https://github.com/coder/code-server/issues/5019?

Reply:
I am not seeing a `.gitignore` in the `typescript-language-features` extension so probably not unfortunately :cry: 

Will comment more on that issue; planning on getting through all my GitHub notifs today